ui/raidboss/data/05-shb/trial/varis-ex.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+[{
+  zoneRegex: {
+    en: /^Memoria Misera \(Extreme\)$/,
+  },
+  timelineFile: 'varis-ex.txt',
+  triggers: [
+    {
+      id: 'VarisEx Phase 2',
+      regex: Regexes.startsUsing({ source: 'Varis Yae Galvus', id: '4CCC', capture: false }),
+      run: function(data) {
+        data.phase = 'phase2';
+      },
+    },
+    {
+      id: 'VarisEx Clones',
+      regex: Regexes.ability({ source: 'Phantom Varis', id: '4CB3', capture: false }),
+      run: function(data) {
+        data.clones = 'active';
+      },
+    },
+    {
+      id: 'VarisEx Clones Cleanup',
+      regex: Regexes.ability({ source: 'Phantom Varis', id: '4CB3', capture: false }),
+      delaySeconds: 20,
+      run: function(data) {
+        delete data.clones;
```

Comment:
It looks like the `clones` property is used only for the dodge trigger? If so, it might be better to move the `delete` logic to the end of the dodge trigger, rather than having an entirely separate trigger for cleanup.